src/Kovan-DssSpell.sol

```diff
@@ -38,129 +28,33 @@ contract SpellAction {
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/kovan/1.1.4/contracts.json
 
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+    ChainlogAbstract constant CHANGELOG     = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
-    address constant FLIP_FAB        = 0x7c890e1e492FDDA9096353D155eE1B26C1656a62;
+    address constant MCD_ADM                = 0x27E0c9567729Ea6e3241DE74B3dE499b7ddd3fe6;
+    address constant VOTE_PROXY_FACTORY     = 0xEec40383cbeE179Be76D4D930c0ABD0D4beA672f;
 
-    address constant GUSD            = 0x31D8EdbF6F33ef858c80d68D06Ec83f33c2aA150;
-    address constant MCD_JOIN_GUSD_A = 0x0c6B26e6AB583D2e4528034037F74842ea988909;
-    address constant MCD_FLIP_GUSD_A = 0xf6c0e36a76F2B9F7Bd568155F3fDc53ff1be1Aeb;
-    address constant PIP_GUSD        = 0xb6630DE6Eda0f3f3d96Db4639914565d6b82CfEF;
+    function execute() external {
+        address MCD_PAUSE    = CHANGELOG.getAddress("MCD_PAUSE");
+        address FLIPPER_MOM  = CHANGELOG.getAddress("FLIPPER_MOM");
+        address OSM_MOM      = CHANGELOG.getAddress("OSM_MOM");
 
-    uint256 constant THOUSAND = 10**3;
-    uint256 constant MILLION = 10**6;
-    uint256 constant WAD = 10**18;
-    uint256 constant RAY = 10**27;
-    uint256 constant RAD = 10**45;
+        // Change MCD_ADM address in the changelog (Chief)
+        CHANGELOG.setAddress("MCD_ADM", MCD_ADM);
 
-    // Many of the settings that change weekly rely on the rate accumulator
-    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
-    // To check this yourself, use the following rate calculation (example 8%):
-    //
-    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
-    //
-    uint256 constant FOUR_PERCENT_RATE = 1000000001243680656318820312;
+        // Add VOTE_PROXY_FACTORY to the changelog (previous one was missing)
+        CHANGELOG.setAddress("VOTE_PROXY_FACTORY", 0xEec40383cbeE179Be76D4D930c0ABD0D4beA672f);
```

Comment:
the address is already a constant here `VOTE_PROXY_FACTORY`

Reply:
Changed!